### PR TITLE
add support for using netCDF-Fortran as dependency in ALADIN easyblock (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyblocks/a/aladin.py [hunk 1]
```diff
@@ -181,10 +182,22 @@ def configure_step(self):
...
-        netcdfinc = os.path.join(get_software_root('netCDF'), 'include')
         mpilib = os.path.join(os.getenv('MPI_LIB_DIR'), os.getenv('MPI_LIB_SHARED'))
 
+        # netCDF
+        netcdf = get_software_root('netCDF')
+        netcdf_fortran = get_software_root('netCDF-Fortran')
+        if netcdf:
+            netcdfinc = os.path.join(netcdf, 'include')
+            if netcdf_fortran:
+                netcdflib = os.path.join(netcdf_fortran, get_software_libdir('netCDF-Fortran'), 'libnetcdff.a')
```
Comment (wpoely86) [inline easybuild/easyblocks/a/aladin.py]: `get_software_libdir` doesn't give a full path?

Comment (boegel) [inline easybuild/easyblocks/a/aladin.py]: nope, it doesn't

**Comment by boegel** [comment]
tested with https://github.com/hpcugent/easybuild-easyconfigs/pull/2207, works as intended

**Comment by boegel** [comment]
going in, thanks for the review @wpoely86!

**Pull request merged**